keyboards/hp69/keymaps/via/keymap.c

```diff
@@ -46,4 +46,17 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     ),
 
 
+};
+
+
+void matrix_scan_user(void) {
+    uint8_t layer = biton32(layer_state);
+    switch (layer) {
+        case 0:
+            writePinLow(A7);
+            break;
+        case 1:
+            writePinHigh(A7);
+            break;
+    }
```

Comment:
This could be simplified to just: 
```suggestion
void matrix_scan_user(void) {
	writePin(A7, layer_state_is(1));
```